SalesforceSDK/Salesforce.Sample.HybridMainPage.Universal/ApplicationInsights.config

```diff
@@ -0,0 +1,23 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I didn't consider that scenario, I will test it once I get VS2015 installed on my windows 8.1 machine.